includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -1143,4 +1175,74 @@ public function get_content( int $post_id = 0, int $return_length = 0, bool $use
 		return apply_filters( 'classifai_chatgpt_content', $content, $post_id );
 	}
 
+	/**
+	 * Sanitize the prompt data.
+	 * This is used for the repeater field.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array $prompts Prompt data.
+	 *
+	 * @return array Sanitized prompt data.
+	 */
+	public function sanitize_prompts( array $prompts ): array {
+		// Remove any prompts that don't have a title and prompt.
+		$prompts = array_filter(
+			$prompts,
+			function ( $prompt ) {
+				return ! empty( $prompt['title'] ) && ! empty( $prompt['prompt'] );
+			}
+		);
+
+		// Sanitize the prompts and make sure only one prompt is marked as default.
+		$has_default = false;
+
+		return array_map(
+			function ( $prompt ) use ( &$has_default ) {
+				$default = $prompt['default'] && ! $has_default;
+
+				if ( $default ) {
+					$has_default = true;
+				}
+
+				return array(
+					'title'   => sanitize_text_field( $prompt['title'] ),
+					'prompt'  => sanitize_textarea_field( $prompt['prompt'] ),
+					'default' => absint( $default ),
+				);
+			},
+			$prompts
+		);
+	}
+
+	/**
+	 * Get the default prompt for use.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array $prompts Prompt data.
+	 *
+	 * @return string|null Default prompt.
+	 */
+	public function get_default_prompt( array $prompts ) {
+		$excerpt_prompt = null;
```

Comment:
Seems this is used for all prompts, not just the excerpt prompt. Probably want to make this variable name more generic.

Reply:
Make sense. I will use `$default_prompt`.